### DISCUSSION TITLE
TTGO Lillygo T-WATCH 2020 display ESP32

### DISCUSSION THREAD

**Discussion by rarenu** [HASwitchPlate/openHASP#424] [Hardware]
Dear sir, I request you to consider supporting TTGO Lillygo T-Watch 2020 ESP32, display so that the Home Automation comes to Wearable level. The above watch has ESP32 processor, touchscreen and memory requirements to convert it into a OpenHASP.
Please think it over and give us a good news soon. This is going to be a turning point in the Home Automation world. Thank you in advance.

**Comment by fvanroie** [comment]
Hi there. Do you own such a device? I am curious on what the expected battery life of the T-WATCH is.
One of the challenges will be to get a decent amount of autonomy or it would not be very useful.

**Reply by rarenu** [reply]
Hi, thank you for your reply. My brother has one and the battery lasts about 20~24 hours depending on the usage. My order is on the way and reaching me in a weeks time.
Please refer the attached pdf file for more information on the product.
Thank you again and Wish You All The Best.
[Lilygo® Ttgo T-watch 2020 V3 Esp32 Programmable Watch.pdf](https://github.com/HASwitchPlate/openHASP/files/10701978/Lilygo.Ttgo.T-watch.2020.V3.Esp32.Programmable.Watch.pdf)

**Reply by rarenu** [reply]
Hi, please go through the product details and explore the links below:
https://github.com/Xinyuan-LilyGO/TTGO_TWatch_Library
https://github.com/sharandac/My-TTGO-Watch
If you find it interesting and wanted pursue the development, let me know I will gift you one "LILYGO® TTGO T-Watch 2020 V3 ESP32 Programmable Watch Microphone IPS Touchable With WIFI Bluetooth Vibration Motor Speaker". 
Please reply your interest.

**Comment by rarenu** [comment]
https://youtu.be/b_bIUH-4Auo

https://youtube.com/shorts/b_bIUH-4Auo?feature=share
